e2e/config.yaml.sample

```diff
@@ -24,9 +24,20 @@ pvcspecs:
     storageclassname: rook-cephfs-fs1
     accessmodes: ReadWriteMany
 
+# List of Deployer specifications.
+# These define the configuration values for different deployers.
+# Available deployers: appset, subscr, disapp
+deployers:
+  - name: appset
+    type: appset
+  - name: subscr
+    type: subscr
+  - name: disapp
+    type: disapp
```

Comment:
This change is also needed in ramenctl:
https://github.com/RamenDR/ramenctl/blob/1f58857d9d5bb4e3c5a8cb2f670e64975243cbba/pkg/config/sample.yaml#L32